cfme/tests/configure/test_default_views.py

```diff
@@ -84,3 +101,33 @@ def test_grid_defaultview(request, setup_a_provider, key):
     sel.force_navigate(name[1])
     assert tb.is_vms_grid_view(), "Grid Default view setting failed"
     reset_default_view(name[0], default_view)
+
+
+@pytest.mark.parametrize('key',
+    cfme_data.get('defaultview_data', {})['exp_comp']['infra'], scope="module")
+def test_expanded_view(request, setup_a_provider, key):
+    name = re.split(r"\/", key)
+    default_view = get_default_view(name[0])
+    set_expanded_view(name[0])
+    sel.force_navigate(name[1])
+    Quadicon.select_first_quad()
+    checkbox = ("(.//input[@id='listcheckbox'])[2]")
```

Comment:
Is this the `Quadicon` checkbox? Since this is a common element to all of these tests, I'd love to see it be modeled somewhere outside of these tests.